src/Services/RegisterService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once 'src/models/UsersModel.php';
+class RegisterService
+{
+    static public function validateUsername(string $username): bool
+    {
+        if (strlen($username) < 3 || strlen($username) > 25)
+        {
+            return false;
+        }
+
+        return true;
+    }
+
+    static public function validateEmail(string $email): bool
+    {
+        if (!filter_var($email, FILTER_VALIDATE_EMAIL))
+        {
+            return false;
+        }
+
+        return true;
+    }
+
+    static public function validatePassword(string $password): bool
+    {
+        if (strlen($password) < 8 || strlen($password) > 25)
```

Comment:
The password field in your database goes up to 50 characters, why are you limiting it to 25 here?